tests/languages/elixir/capture_feature.test

```diff
@@ -7,7 +7,12 @@ fun = &(&1 + 1)
 
 [
 	"fun ", ["operator", "="],
-	["capture", "&Math.zero?/1"],

```

Comment:
What do we do about captures? Having `&Math.zero?/1` and `&is_function/1` be highlighted differently is quite inconsistent.

Reply:
I've updated the `function` tokenizer and removed capture as it was a subset of function